### PR TITLE
Add subscription default payment instrument

### PR DESCRIPTION
**Pull request by VladimirBerdnik**

### CODE REVIEW

**Review by n-fasano — DISMISSED** [review]

**Review by nanobash — DISMISSED** [review]

File: openapi/components/schemas/Common/CommonSubscriptionOrder.yaml [hunk 1]
```diff
@@ -129,3 +129,10 @@ allOf:
             x-sortable: true
             format: double
             example: 49.95
+      paymentInstrumentId:
+        description: >-
+          The payment instrument ID to use for autopay.
```
Comment (noon-dawg) [inline openapi/components/schemas/Common/CommonSubscriptionOrder.yaml:134]:
```suggestion
          ID of the payment instrument to use for autopay.
```

File: openapi/components/schemas/Common/CommonSubscriptionOrder.yaml [hunk 2]
```diff
@@ -129,3 +129,10 @@ allOf:
...
+          If not provided or inactive, the customer default payment instrument will be used.
```
Comment (noon-dawg) [inline openapi/components/schemas/Common/CommonSubscriptionOrder.yaml:135]:
```suggestion
          If not provided or inactive, the customer's default payment instrument will be used.
```

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/Common/CommonSubscriptionOrder.yaml [hunk 3]
```diff
@@ -129,3 +129,10 @@ allOf:
             x-sortable: true
             format: double
             example: 49.95
+      paymentInstrumentId:
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonSubscriptionOrder.yaml:132]: Are you also adding it to Storefront resource?

https://preview.redoc.ly/rebilly-storefront/Subscription-default-payment-instrument/tag/Orders#operation/StorefrontGetOrder!c=200&path=0/paymentInstrumentId&t=response

Comment (VladimirBerdnik) [reply openapi/components/schemas/Common/CommonSubscriptionOrder.yaml:132]: Yes, as soon as Storefront API has an endpoint to get list of payment instruments

**Review by ashkarpetin — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Pull request merged**